media-autobuild_suite.bat

```diff
@@ -1624,8 +1624,8 @@ if %msys2%==msys32 (
 :: write config profiles:
 ::------------------------------------------------------------------
 
-if %build32%==yes call :writeProfile 32
-if %build64%==yes call :writeProfile 64
+if %build32%==yes if NOT EXIST %instdir%\local32\etc\profile2.local call :writeProfile 32
```

Comment:
Maybe move this to inside the writeProfile call?

Also, I'd prefer not having this check, since if we do need to change the contents of profile2.local we'd need to change its name again, but maybe that's the cleaner/more efficient way.

Reply:
The main reason why I had added this was because if someone wanted to change the environment variables, such  as using clang instead of GCC, through the profile2.local, it would get overwritten.

Maybe instead use something like profile.d and just drop a text file with personalize stuff to override profile2? What do you think?

Reply:
nvm.
```


/local32|64/etc/custom_profile & $HOME/custom_build_options

    Put here any general/platform tweaks that you need for your specific environment. See /local32|64/etc/profile2.local for example usage.
```